boards/arm/mimxrt595_evk/mimxrt595_evk_cm33_defconfig

```diff
@@ -13,6 +13,7 @@ CONFIG_SERIAL=y
 CONFIG_PINCTRL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
+CONFIG_I2S=y
```

Comment:
should this be in board defconfig? it's not necessary for basic functioning of the board